code/scripting/api/libs/ui.cpp

```diff
@@ -885,40 +885,57 @@ ADE_FUNC(canSkip,
 
 ADE_FUNC(replayMission,
 	l_UserInterface_Debrief,
+	"[boolean restart = false]",
```

Comment:
This should say "restart = true" to match the default

Reply:
Right. Done.